bool flag_detect
---
geometry_msgs/Point grasp_center
geometry_msgs/Quaternion grasp_ori
